package.json

```diff
@@ -190,6 +190,7 @@
 		"config": "^3.3.6",
 		"debug": "^4.3.3",
 		"enhanced-resolve": "5.9.3",
+		"framer-motion": "^6.2.8",
```

Comment:
@arthur791004 CMIIW, but I don't think we need to add `framer-motion` to `package.json` and `yarn.lock` anymore. Instead, we will need to add it to `packages/components/package.json`.

Reply:
It's true for the `package.json`. But we still needs to update the `yarn.lock` via `yarn install` automatically

Reply:
Since `@automattic/components` is a fairly commonly used package, I'm concerned that introducing `framer-motion`  would unnecessarily increase the bundle of other parts in Calypso. Should we create a new package like `@automattic/animation-frame-motion` or is that overkill?

Reply:
I'll update and rerun the iscalypsofastyet again to see the affection